includes/data-stores/abstract-wc-order-item-type-data-store.php

```diff
@@ -59,11 +59,15 @@ public function create( &$item ) {
 	public function update( &$item ) {
 		global $wpdb;
 
-		$wpdb->update( $wpdb->prefix . 'woocommerce_order_items', array(
-			'order_item_name' => $item->get_name(),
-			'order_item_type' => $item->get_type(),
-			'order_id'        => $item->get_order_id(),
-		), array( 'order_item_id' => $item->get_id() ) );
+		$changes = $item->get_changes();
+
+		if ( ! empty( $changes ) ) {
```

Comment:
@mikejolley I'm wondering if 55ba115ac535493abd6ab6d08dd689afdef82511 would be enough? There is no need to check `order_item_type` because `get_type` *always* return the same string (it's _hardcoded_) as far as I could see. Let me know if anything else is missing.